bears-router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
+const db = knex(knexConfig);
+
+router.post('/', (req,res) => {
+    if (!req.body.name) {
+        res.status(400).json({error:'missing required field: name'});
+    } else {
+    db('bears').insert(req.body)
+        .then(ids => {
+            const id = ids[0];
+            db('bears').where({id: id}).first()
```

Comment:
you can shorten this where statement even further based on the identical property-value pair that ES6 allows us to do: `where({id})`